packages/Select/Select.jsx

```diff
@@ -131,18 +131,21 @@ const renderError = (error, errorId) => (
   </InputFeedback>
 )
 
-const renderLabel = (id, label, hint, hintId, tooltip) => (
-  <Box inline between="space-between">
-    <label htmlFor={id || generateId(label).identity()}>
-      <StyledLabelContainer inline tag="span" between={2}>
-        <Text size="medium" bold data-testid="selectLabel">
-          {label}
-        </Text>
-        {hint && renderHint(hint, Text, hintId)}
-      </StyledLabelContainer>
-    </label>
-    {tooltip && React.cloneElement(tooltip, { connectedFieldLabel: label })}
-  </Box>
+const renderLabel = (id, label, hint, hintPosition, hintId, tooltip) => (
+  <div>
```

Comment:
Can this `<div>` be replaced with Fragment?

Reply:
No, it needs to use the `div` so as to not be affected by the `Box`